public/js/app.js

```diff
@@ -112,6 +112,23 @@ jQuery( document ).ready(function( $ ){
                 }
             });
         });
+
+        $('#hireme').click(function() {
+            var data = {
+                _token: $('#_token').val();,
+                message: $('#message').val();,
```

Comment:
Missing semicolon.<br>Expected '}' to match '{' from line 116 and instead saw ':'.<br>Expected ')' and instead saw '$'.<br>Expected an identifier and instead saw ','.<br>Expected an assignment or function call and instead saw an expression.